README.md

```diff
@@ -49,17 +50,13 @@ These reside in the [integration features directory](features/integration)
 | ---------------------- | -------------------------------------------------------------------------------------- |
 | @abi                   | Test the Application Binary Interface (ABI) with atomic txn composition and execution. |
 | @algod                 | General tests against algod REST endpoints.                                            |
-| @application.evaldelta | Test that eval delta fields are included in algod and indexer.                         |
 | @applications.verified | Submit all types of application transactions and verify account state.                 |
 | @assets                | Submit all types of asset transactions.                                                |
 | @auction               | Encode and decode bids for an auction.                                                 |
 | @c2c                   | Test Contract to Contract invocations and injestion.                                   |
 | @compile               | Test the algod compile endpoint.                                                       |
 | @dryrun                | Test the algod dryrun endpoint.                                                        |
 | @dryrun.testing        | Test the testing harness that relies on dryrun endpoint. Python only.                  |
-| @indexer               | Test all types of indexer queries and parameters against a static dataset.             |
-| @indexer.231           | REST Client Integration Tests for Indexer 2.3.1+                                       |
-| @indexer.applications  | Endpoints and parameters added to support applications.                                |
```

Comment:
Propose the removal of all indexer features which only rely on hard-coded database fixtures.

Reply:
@winder ^^^

Reply:
I think the cucumber unit tests cover all of these things, so I'm fine with removing them. They tested Indexer more than anything else, but that doesn't help find regressions since the versions are pinned.